packages/elements/src/react/common/form/index.tsx

```diff
@@ -170,9 +179,12 @@ const useInput = ({
   onChange: onChangeProp,
   onBlur: onBlurProp,
   onFocus: onFocusProp,
+  // @ts-expect-error - Depending on type the props can be different
+  passkeyAutofill,
```

Comment:
If you look at `validatePassword` (and e.g. line 241) you'll see the same problem.

I chose to not add the optional prop to the destructuring here but instead typecast the `passthroughProps` and then get `validatePassword` from there.

Then you need to typecast only once and don't need to add ts-expect-error in multiple places. Makes more sense to me 🤔